cms-adapter/src/Rsbc.Dmf.CaseManagement.Service/Services/CaseService.cs

```diff
@@ -853,12 +853,22 @@ public async override Task<ResultStatusReply> CreateBringForward(BringForwardReq
         {
             ResultStatusReply reply = new ResultStatusReply();
 
-            // fetch the document.
             try
             {
+                var bringForwardRequest = new CaseManagement.BringForwardRequest()
+                {
+                    CaseId = request.CaseId ?? string.Empty,
+                    Assignee = request.Assignee ?? string.Empty,
+                    Description = request.Description?? string.Empty,                
+                    Subject = "ICBC Error",
+                    Priority = (CaseManagement.BringForwardPriority?)BringForwardPriority.Normal
+                };
+                
                 // call _caseManager...
+                await _caseManager.CreateBringForward(bringForwardRequest);
 
-                throw new Exception();
+                reply.ResultStatus = ResultStatus.Success;
```

Comment:
It would be better for this to set the result status based on the result of CreateBringForward - if it is true, set it to success.